backend/app/models/InvitationResult.scala

```diff
@@ -23,8 +23,13 @@ package models
 
 import play.api.libs.json.{Format, Json}
 
-case class InvitationLink(id: InvitationId, email: String)
+/** @param invitationLinkId
+  *   is optional and only provided, if a link was created.
```

Comment:
looks odd, how about
```suggestion
/** @param invitationLinkId is optional and only provided, if a link was created.
```

Reply:
auto-reformatted due to scalafmt...